components/splitbutton/stories/template.js

```diff
@@ -3,7 +3,7 @@ import { classMap } from "lit/directives/class-map.js";
 
 import { Template as Button } from "@spectrum-css/button/stories/template.js";
 
-import "../index.css";
+import "@spectrum-css/splitbutton/index.css";
```

Comment:
I wonder if we need to point to dist/index-vars.css here to have access to the global dimensions?

Reply:
I have tested a few variations of linking to the package and linking to dist seems to be a no-go. For how we've go our stories processing, we want to reference the pre-compiled asset here.